plugin/manager_linux.go

```diff
@@ -132,26 +131,32 @@ func (pm *Manager) restore(p *v2.Plugin) error {
 		return pm.pluginPostStart(p, c)
 	}
 
+	if alive {
+		// TODO(@cpuguy83): Should we always just re-attach to the running plugin instead of doing this?
```

Comment:
remove this

Reply:
Why? This is a legitimate todo item.

Reply:
Hm. Plugins should be best-effort consistent with container behavior. In that respect, I dont think we should try re-attach.